tests/integration/tei/test_tei_exporting.py

```diff
@@ -0,0 +1,19 @@
+"""Integration tests for the TEI exporter."""
+
+import pytest
+
+from inspirehep.modules.hal import tei
+from inspirehep.utils.record_getter import get_db_record
+
+SAMPLE = "tests/integration/tei/fixtures/sample.xml"
+
+
+def test_format_tei(app):
+    r = get_db_record('literature', 1407863)
+
+    expected = None
+    with open(SAMPLE, "r") as output:
```

Comment:
Time to read about `setuptools`, in particular `pkg_resources`: http://setuptools.readthedocs.io/en/latest/pkg_resources.html